robocup_knowledge/src/robocup_knowledge/environments/rgo2019/challenge_where_is_this.py

```diff
@@ -0,0 +1,48 @@
+# RIPS KNOWLEDGE FILE RGO2019
+
+# TU/e Robotics
+from robocup_knowledge import knowledge_loader
+
+# Common knowledge
+common = knowledge_loader.load_knowledge("common")
+
+grammar_target = "T"
+
+location_grammar = """
+T[A] -> COURTESY_PREFIX VP[A] | VP[A]
+
+COURTESY_PREFIX -> robot please | could you | could you please | please
+"""
+
+for loc in common.location_rooms:
+    location_grammar += '\nLOCATION[{"id": "%s"}] -> %s' % (loc, loc)
+
+for loc in common.location_names:
```

Comment:
These lines are double

Reply:
I do think there's a difference between ```location_rooms``` and ```location_names``` ;)

Reply:
I will have my glasses checked ;P